services/datalad/datalad_service/common/github.py

```diff
@@ -23,17 +23,30 @@ def create_github_repo(dataset_path, dataset_id):
             'DATALAD_GITHUB_TOKEN and DATALAD_GITHUB_ORG must be defined to create remote repos')
 
 
-def github_export(dataset_path, tag):
+def github_export(dataset_id, dataset_path, tag):
     """
     Publish GitHub repo and tags.
     """
     subprocess.check_call(
         ['git', 'push', 'github', f'{tag}:refs/heads/main'], cwd=dataset_path)
+    subprocess.check_call(
+        ['git', 'push', 'github', f'{tag}:refs/heads/master'], cwd=dataset_path)
     subprocess.check_call(
         ['git', 'push', 'github', 'git-annex:refs/heads/git-annex'], cwd=dataset_path)
     # Update tags
     subprocess.check_call(
         ['git', 'push', '--tags', 'github'], cwd=dataset_path)
+    # Make sure the default branch is correct
+    github_set_default_branch(dataset_id)
+
+
+def github_set_default_branch(dataset_id):
+    """Sets the repo default branch to 'main' and resets the description/homepage fields."""
+    ses = Github(DATALAD_GITHUB_TOKEN)
+    org = ses.get_organization(DATALAD_GITHUB_ORG)
+    repo = org.get_repo(dataset_id)
+    repo.edit(default_branch="main", description="OpenNeuro dataset",
```

Comment:
Is it reasonably un-clunky to put the dataset name here? That would make the listing in https://github.com/OpenNeuroDatasets much more informative.

```suggestion
    repo.edit(default_branch="main", description=f"OpenNeuro dataset {dataset_id}: {description['Name']}",
```

Reply:
As part of the changes for metadata tagging on GitHub I was planning on loading some information from OpenNeuro's API service and updating it here.

That said, we could just read the dataset_description.json inline here and use the name from that if it exists, not sure how soon we'll get back to those GitHub metadata changes with other priorities.